packages/app/src/app/pages/index.tsx

```diff
@@ -221,6 +221,7 @@ const RoutesComponent: React.FC = () => {
             <Route path="/standalone/:componentId" component={StandalonePage} />
             <Route path="/create-workspace" component={CreateWorkspace} />
             <Route path="/upgrade" component={UpgradeWorkspace} />
+            <Route path="/signup-beta" component={SignUpUBB} />
```

Comment:
Is there room for changing this URL? I think sign up might be misleading to a sort of authentication process instead of a plan. What about `/workspace-new`, `/workspace-beta`?

Reply:
This is a temporary URL only and will be shared directly to people that want to subscribe to the new ubb models. I think this is the wording used in the banners that advertised ubb before the release. cc @necoline, this is an easy change, so whichever URL format works best we can deploy that to prod today